### ISSUE TITLE
Profile Picture Desynchronization

### ISSUE THREAD

**Issue by TGRush** [oxen-io/session-android#1204]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**

It commonly happens that Profile Pictures are out-of-date with linked devices that haven't been used in a while, or when linking a new device.

This causes two things:
1. Already linked devices have an old profile picture
2. Newly linked devices will not have any profile pictures - including your own - AT ALL.

**To reproduce**

1. Link a new device
2. Profile Pictures are missing

OR

1. Check a linked device which hasn't been in use for a while
2. Profile picture is outdated / desynchronized

**Smartphone (please complete the following information):**

 - Device: Pocophone F3
 - OS: Android 12L
 - Version of Session: 1.16.7 (3355)

**Additional context**

I have observed this issue on all devices which are able to access session. My iPad Air 2 (15.7.1) and my Linux Desktop both showed this behavior.

**Proposed solution:**

I propose one of two things:

1. **A "send profile picture now" button.**
This is a button in Threema which allows you to send a contact your up-to-date profile picture. It wouldn't solve the issue for your own profile picture on newly linked devices.
2. **Requesting Profile Pictures from all contacts retrieved from the swarm upon linking a new device.**
Self Explanatory.

Note that this doesn't happen for communities, they serve their profile pictures just fine every time a device is linked.

**Comment by KeeJef** [comment]
Yeah we are aware of bugs in the current profile image syncing code, this should be addressed in the upcoming user config release, PFP's will sync much more reliably with the new release
